index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <a href="#"><img src="images/menu_icon.png" alt="menu" /></a>
```

Comment:
- [ ] You have implemented the hamburger menu in a good way. Let's put it in a nav tag to improve our structure and semantics.